manifests/init.pp

```diff
@@ -96,12 +96,12 @@
     if ! has_key($ajp, 'port') {
```

Comment:
has_key is deprecated if you wanted to fix this too (use `in`). Probably also use `unless`?

Reply:
thanks, I forgot hat has_key() is deprecated as well. I will fix that in another PR.